.github/workflows/ci.yml

```diff
@@ -10,16 +10,20 @@ on:
     # can add tags if needed
 
 jobs: 
-    test: 
-        name: Test
-        runs-on: ubuntu-latest
+    test:
+        name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                version: ['1.7.2']
```

Comment:
@ShouvikGhosh2048 here can you change the version to just `['1']`? I think our recent tests are in `1.8.x`, and this might be making the tests fail.

Reply:
Ubuntu fails on 1

Reply:
Is it possible to run the test on version `1.8.x`?

Reply:
@sourish-cmi Yes, with version being `['1']` the CI will get the latest version of Julia. The most recent test was done in `1.8.5`: https://github.com/xKDR/CRRao.jl/actions/runs/3978462222/jobs/6820258360

Reply:
okay